README.md

```diff
@@ -173,55 +178,62 @@ NSManagedObjectContext *backgroundContext = [[VOKCoreDataManager sharedInstance]
 [backgroundContext performBlock:^{
     SomeManagedObjectSubclass *thing = [SomeManagedObjectSubclass vok_newInstanceWithContext:backgroundContext];
     thing.someArbitrayAttribute = @"hello";
-    [[VOKCoreDataManager sharedInstance] saveAndMergeWithMainContext:backgroundContext];  
+    [[VOKCoreDataManager sharedInstance] saveAndMergeWithMainContext:backgroundContext];
 }];
 ```
 
 **NOTE**: Temporary contexts created manually or vended through the convenience background methods are child contexts of the main context.
 
 ###Inserting records
-
 ```objective-c
 VOKPerson *person = [VOKPerson vok_newInstance];
-[person setFirstName:@"Rohan"];
-[person setLastName:@"Panchal"];
+person.firstName = @"Rohan";
+person.lastName = @"Panchal";
 [[VOKCoreDataManager sharedInstance] saveMainContextAndWait];
 ```
 
-###Querying Records	
+###Querying Records
+Like the example above, this one makes use of the `VOKKeyForInstanceOf` macro from [VOKUtilities/VOKKeyPathHelper](https://github.com/vokal/VOKUtilities#vokkeypathhelper) for the keys in these calls to ensure that typos like `@"lsatName"` don't occur.
 
 ####Query with basic predicate
 ```objective-c
-NSArray *results = [VOKPerson vok_fetchAllForPredicate:nil forManagedObjectContext:nil]; //Basic Fetch
+    NSPredicate *smithsPredicate = [NSPredicate predicateWithFormat:@"%K == %@"
+                                                      argumentArray:@[VOKKeyForInstanceOf([VOKPerson class], lastName), @"Smith"]];
```

Comment:
1. Should be `VOKKeyForInstanceOf(VOKPerson, lastName)`.  The macro calls `class` for you, (also below).
1. Any reason to prefer using `argumentsArray` instead of var args?